src/twisted/conch/ssh/keys.py

```diff
@@ -382,7 +382,7 @@ def _fromString_PRIVATE_OPENSSH(cls, data, passphrase):
             except ValueError:
                 raise BadKeyError('invalid DEK-info %r' % (lines[2],))
 
-            if cipher == b'AES-128-CBC':
+            if cipher in (b'AES-128-CBC', b'AES-256-CBC'):
```

Comment:
`AES-256-CBC` is required here because `serialization.BestAvailableEncryption` uses `AES-256-CBC` now.